fvtest/jitbuildertest/SystemLinkageTest.cpp

```diff
@@ -53,9 +69,19 @@ class SystemLinkageTest : public JitBuilderTest {};
 
 TEST_F(SystemLinkageTest, FooTest)
    {
+#if defined(PPC)
+   // TODO (#4765): Fix the Power linkage and enable this test
+   // TODO (#4764): This is a poor man's SKIP_IF which is only available for Tril tests currently. We should find a
+   // nice home for this useful utility so we can use it in JitBuilder tests as well.
+   const ::testing::TestInfo* const test_info = ::testing::UnitTest::GetInstance()->current_test_info();
+   ::testing::Test::RecordProperty("skipped", "Known Bug");
+   std::cout << "Known Bug" << ": Skipping test: " << test_info->name() << "\n    " << "Power system linkage cannot handle stack arguments (see issue #4765)" << "\n";
+   SUCCEED() << "Power system linkage cannot handle stack arguments (see issue #4765)";
+#endif
```

Comment:
This is another hack we are adding in, but I don't think this PR is the correct place to solve this. I've opened up two new issues to address the hack added here.